DirectusExtensions/charging-stations-bundle/src/hook-on-create-charging-station/index.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
+
+import { defineHook } from '@directus/extensions-sdk';
+
+export default defineHook(({ action }, { env }) => {
+
+    action('ChargingStations.items.create', (input) => {
+        console.log("Subscribing " + input.key + " to connect and close events");
+        
+        const stationId = input.key;
+        const subscriptionUrl = `${env.CITRINEOS_URL}${env.CITRINEOS_SUBSCRIPTION_API_PATH}`;
+        const updateStationStatusUrl = `${env.DIRECTUS_URL}${env.DIRECTUS_CHARGING_STATION_UPDATE_STATUS_PATH}`;
```

Comment:
perhaps we can connect, if you are calling an endpoint to update the DB, then there may be a better approach where you can make DB queries directly from here

Reply:
this is already running on Directus BE service, why would you then call another directus BE service to then perform the logic rather then handing what you need here?

Reply:
When you review the code, you'll see that the directus url is not called here. It is given to CitrineOS's subscription api, which then calls it when there are relevant updates.

Reply:
I think we should have a quick discussion here - in this case I was talking about line 14 which is for the `updateStationStatusUrl`